### DISCUSSION TITLE
[Urgent response required] Unable to generate the QR code for the past few days already

### DISCUSSION THREAD

**Discussion by syahir10syahir** [wppconnect-team/wppconnect#2404] [Q&A]
Hi,
we are unable to generate the QR code - we have tried with fresh installation also and having the same - it keep saying the status "    "status": "INITIALIZING","

here is the response 

<img width="611" alt="image" src="https://github.com/user-attachments/assets/fca78471-1912-4bce-8bb1-335a9a6ab425">

**Comment by manfe** [comment]
Update whatsapp version.
